javascript/javascript1/week1/pizza.js

```diff
@@ -0,0 +1,24 @@
+console.log("I love pizza");
+const favPizza = "Margarita";
+const pizzaPrice = 100;
+console.log(`New Pizza Order: ${favPizza}`);
+console.log(`The Price of the Pizza is: ${pizzaPrice}`);
+
+// 2part of exersice
+
+const NumPizza = 3;
+let strFamily;
```

Comment:
I'm not sure what this variable name means. Consider using a more descriptive one.